airflow/providers/google/CHANGELOG.rst

```diff
@@ -24,129 +24,147 @@ Changelog
 Breaking changes
 ~~~~~~~~~~~~~~~~
 
-* ``apache-airflow-providers-google uses deprecated Google Ads API V8 (#22111)``
```

Comment:
Mmm this is an entry that should stay...

Reply:
:facepalm:  Glad you are checking